R/args.R

```diff
@@ -147,15 +149,38 @@ SampleArgs <- R6::R6Class(
       self$thin <- thin
       self$max_depth <- max_depth
       self$metric <- metric
-      # self$metric_file <- character()
+      self$inv_metric <- inv_metric
+      if(!is.null(inv_metric)) {
+        if(!is.null(metric_file)) {
+          stop("Only one of inv_metric and metric_file can be non-NULL")
```

Comment:
```r
stop("Only one of inv_metric and metric_file can be specified.", call. = FALSE)
```
Are you ok with that?

Reply:
Well it kinda sucks from an interface perspective. We could try to do a combined argument, but I just didn't want to call an argument where you're passing in the inverse metric by value metric_file.

Reply:
I was actually just suggesting changing the error message to 

```r
stop("Only one of inv_metric and metric_file can be specified.", call. = FALSE)
```

but I agree that it's not ideal from an interface perspective. (super weird that CmdStan uses `metric_file` for specifying `inv_metric`)

Reply:
Oh lol whoops. Yeah I'll change that.